domain/src/main/java/ru/startandroid/domain/placesmodels/citiesdetailsmodels/Address_components.kt

```diff
@@ -0,0 +1,9 @@
+package ru.startandroid.domain.models.placesmodel.citiesdetailsmodels
+
+
+data class Address_components (
```

Comment:
Name of class and variables are not by java convention. Should be without underscores. 
And check other classes, pls.